src/app/tasks/tasks.component.ts

```diff
@@ -55,6 +59,31 @@ export class TasksComponent implements OnInit {
     this.tasksService.addTask({ ...task, assignee }).subscribe((res) => {
       this.tasksService.getTasks();
     });
+    this.sendNotifications();
+  }
+
+  sendNotifications(assignee: any = '') {
+    const link = '/teams/view';
```

Comment:
This link won't show anything.  It needs the team id at the end `/teams/view/${teamID}`